crates/bevy_ui/src/render/mod.rs

```diff
@@ -180,6 +180,7 @@ pub struct ExtractedUiNode {
     /// Ordering: left, top, right, bottom.
     pub border: [f32; 4],
     pub node_type: NodeType,
+    pub antialias: bool,
```

Comment:
The rest of the implementation looks good, but this part seems wrong. As you are using a global resource to set the anti-aliasing, you shouldn't be passing the `antialias` per node but just once. Maybe by having a second `UiAntiAlias` resource in the render world you keep synchronised with main world resource.

Reply:
Also I think it should be more efficient to have a specialized pipeline, like adding:
```
#[derive(Clone, Copy, Hash, PartialEq, Eq)]
pub struct UiPipelineKey {
    pub hdr: bool,
    pub anti_alias: bool,
}
```

That adds a "ANTI_ALIAS" shader def if anti-aliasing is enabled.
Then in `ui.wgsl` you can use `#ifdef ANTI_ALIAS` gates to control if anti-aliasing should be applied.
I think that could be left for a follow up PR though.

Reply:
Done!